tests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if(NOT DEFINED USE_ROS2)
+if(USE_ROS2)
```

Comment:
Isn't the condition here “ if it is not USE_ROS2”?

Reply:
I fixed again https://github.com/isri-aist/TrajectoryCollection/pull/3/commits/a61ba91cf7c9a42684e85226d200d045a78bd73b